torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -302,10 +303,8 @@ def from_hp_to_intx_static(
         zero_point_domain: Optional[ZeroPointDomain] = ZeroPointDomain.INT,
         _layout: Layout = PlainLayout(),
     ):
+        assert zero_point_domain is not None, "zero_point_domain must not be None"
```

Comment:
zero_point_domain should not be Optional in L303 I think

Reply:
also maybe raise ValueError here, that might be more user friendly than assertion

Reply:
Thank you! Made those modifications